src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  flex-shrink: 0;
+  margin-right: 4px;
+}
+
+.stars--0 :last-child,
+.stars--1 :last-child,
+.stars--2 :last-child,
+.stars--3 :last-child,
+.stars--4 :last-child,
+.stars--5 :last-child {
+  margin-right: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 :nth-child(n + 2),
+.stars--2 :nth-child(n + 3),
+.stars--3 :nth-child(n + 4),
+.stars--4 :last-child {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Add this styles by default to `starts__star` class and add active image using modifiers